### PR TITLE
SpecialReportAlt improvements / visual bug fixes

### PR DESCRIPTION
**Pull request by ioannakok**
## What does this change?
Fixes a couple of visual bugs for SpecialReportAlt. See comments for screenshots.

## Does this change need to be reproduced in dotcom-rendering ?

- [X] No. DCR as it has storybook so these bugs were never introduced. See [relevant PR](https://github.com/guardian/dotcom-rendering/pull/6335).
- [ ] Yes (please indicate your plans for DCR Implementation)

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [x] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [x] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/stylesheets/module/facia-garnett/_sublinks.scss [hunk 1]
```diff
@@ -42,6 +42,16 @@
     }
 }
 
+section.fc-container--special-report-alt-palette {
+    .fc-sublinks {
+        .fc-sublink__title {
+            &:before {
+                border-top: 1px solid rgba(118, 118, 118, .3);
```
Comment (ioannakok) [inline static/src/stylesheets/module/facia-garnett/_sublinks.scss:49]: | Before      | After      |
|-------------|------------|
| TODO | image |

File: static/src/stylesheets/module/facia-garnett/_pillars.scss [hunk 2]
```diff
@@ -347,6 +347,37 @@ $pillars: (
...
+
+            &:hover {
+                .fc-item__avatar {
+                    background-color: transparent;
+                }
+
+                .fc-item__timestamp,
+                .fc-trail__count--commentcount {
+                    background-color: transparent;
+                }
```
Comment (ioannakok) [inline static/src/stylesheets/module/facia-garnett/_pillars.scss:365]: | Before      | After      |
|-------------|------------|
| TODO | image|

File: static/src/stylesheets/module/facia-garnett/_pillars.scss [hunk 3]
```diff
@@ -347,6 +347,37 @@ $pillars: (
...
+        }
+    }
+}
+
+
+section.fc-container--has-palette.fc-container--special-report-alt-palette {
+    .fc-item.fc-item--type-comment.fc-item--pillar-news {
+        &.fc-item--type-comment {
+            .fc-item__container.u-faux-block-link--hover {
+                background-color: #e1d9d2;
```
Comment (ioannakok) [inline static/src/stylesheets/module/facia-garnett/_pillars.scss:376]: image

File: static/src/stylesheets/module/facia-garnett/_pillars.scss [hunk 4]
```diff
@@ -347,6 +347,36 @@ $pillars: (
...
+            }
+        }
+    }
+}
+
+section.fc-container--has-palette.fc-container--special-report-alt-palette {
+    .fc-item.fc-item--type-comment.fc-item--pillar-news {
+        &.fc-item--type-comment {
+            .fc-item__container.u-faux-block-link--hover {
+                background-color: darken($special-report-alt-pastel, 5%);
```
Comment (ioannakok) [inline static/src/stylesheets/module/facia-garnett/_pillars.scss:375]: | Before      | After      |
|-------------|------------|
| image | image|

**Review by HarryFischer — APPROVED** [review]
👍

**Pull request merged**